src/main/java/com/google/firebase/internal/FcmOptionsUtil.java

```diff
@@ -0,0 +1,15 @@
+package com.google.firebase.internal;
```

Comment:
Let's move this into the `messaging` package and drop `public` from the class definition. There's really no need for this class to be visible outside the `messaging` package.

Reply:
done